filesystem/android.c

```diff
@@ -15,7 +15,7 @@ GNU General Public License for more details.
 
 #include "port.h"
 
-#if XASH_ANDROID
+#if XASH_ANDROID && XASH_SDL
```

Comment:
There is nothing non-SDL in these functions. Could you either provide stubs on Java side or stub the functions inside `android.c`?